README.md

```diff
@@ -235,14 +235,17 @@ nyc may create artefact directories within the project root, such as:
 
 ## Require additional modules
 
-The `--require` flag can be provided to `nyc` to indicate that additional
-modules should be required in the subprocess collecting coverage:
+The `--require` flag can be provided to `nyc` to indicate that additional modules should be required in the subprocess collecting coverage:
```

Comment:
I'm thinking the example we give should be `nyc --require esm mocha`.  I suggest this as `esm` is not involved in instrumentation, further enforce that babel / typescript projects should use `@istanbuljs/nyc-config-*`.

Not sure it should be mentioned here but the `--require` flag also operates on the main nyc process for use by `--all`.  In situations with `nyc --all true --instrument false` and `babel-plugin-istanbul` setup the `--all` option only works if `--require @babel/register` is passed to nyc.  Passing it to mocha would cause the tests to be instrumented but unloaded sources would not be seen.

Reply:
Added this section!